jest.config.ts

```diff
@@ -18,7 +18,7 @@ const config: Config = {
 	// cacheDirectory: "/tmp/jest_rs",
 
 	// Automatically clear mock calls, instances, contexts and results before every test
-	// clearMocks: true,
+	clearMocks: true,
```

Comment:
We have the restoreMocks enabled, which clear the mock entirely. Do not enable the clearMocks, the restore makes a deeper clean and it is better for controlling the mocks.